docs/AccessingCarePlanData/AccessingCarePlanData-template.md

```diff
@@ -250,8 +250,8 @@ func symptomTrackerViewController(viewController: OCKSymptomTrackerViewControlle
                             quantitySample: sample,
                             quantityStringFormatter: nil,
                             unitStringKeys: [
-                                HKUnit.degreeFahrenheitUnit() : "\u{00B0F}",
-                                HKUnit.degreeCelsiusUnit()    : "\u{00B0C}"
+                                HKUnit.degreeFahrenheitUnit() : "\u{00B0}F", // °F
```

Comment:
Hi @yas375 Thanks for feedback!
I was thinking putting the `°` directly too. 

I think the reason for using escaped hexadecimal is that it's hard to type or copy-paste unicode chars in some environments. 

However, since it's an iOS app (and will be built using Xcode on OS X), everyone should be able to type/display chars.

I'm +1 with using `°` directly.

@umerkhan-apple What do you think?